models/endpoints.models.js

```diff
@@ -0,0 +1,10 @@
+const { readFile } = require('fs/promises')
+
+module.exports = {
+
+  readEndpoints: function() {
+    return readFile(`${__dirname}/../endpoints.json`)
+    .then(JSON.parse)
+  }
+
+}
```

Comment:
You do not need a model for this endpoint - you can either use fs in the controller, or just require in the endpoints.json file directly!